### PR TITLE
0_2_6 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
Add release notes for `0.2.6`. I still have to update after we merge #740, but otherwise this is ready

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]

File: docs/release/release_0_2_6.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Improvements
+************
+- improve release guide (#668)
+- add logo to repo (#674)
+- improve labels painting speed (#684)
+- add example showing mouse drag callbacks (#690)
+- add main window option to screenshots (#722)
+
+Bugfixs
```
Comment (jni) [inline docs/release/release_0_2_6.rst]: Bug fixes

**Review by jni — APPROVED** [review]
@sofroniew minor typo. You might also add a note somewhere that the segfault issue is fixed by vispy?

File: docs/release/release_0_2_6.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_6.rst]: lol - will fix

**Comment by sofroniewn** [comment]
I have added a note about the fix of #576. I will merge once the tests pass and start making release candidates

**Comment by sofroniewn** [comment]
At least one test has passed and this is docs only so i will merge

**Pull request merged**